examples/drivenet/drivenet_train.ipynb

```diff
@@ -0,0 +1,321 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to DriveNet\n",
+    "\n",
+    "DriveNet is a DNN able to drive an agent on a specific route. Among others, DriveNet can do lane-following while respecting traffic lights.\n",
+    "\n",
+    "### Imitation Learning and Its limitations\n",
```

Comment:
Inconsistent casing - do you capitalize all words in titles or just the first? (seems inconsistent across headings).
If you capitalize 'all' -> Limitations, but don't capitalize fill words like its